go/test/endtoend/vtgate/plan_tests/plan_e2e_test.go

```diff
@@ -40,3 +40,23 @@ func TestSelectCases(t *testing.T) {
 		})
 	}
 }
+
+func TestFilterCases(t *testing.T) {
```

Comment:
instead of copy pasting, we should turn these two methods into one, and use something like:

```go
for _, file := range []string{"select_cases.json", "filter_cases.json"} {
  mcmp.Run(name, func(mcmp *utils.MySQLCompare) {
...
  }
}
```

Reply:
Sure, made a new `TestE2ECases` which runs both select_cases and filter_cases test files  with 7ac0eb2c002e364991bcf3b9630ad58be407d16e